common/common.c

```diff
@@ -1503,3 +1503,39 @@ void adjust_visual_label(char *text, int method, int rep_dash) {
     }
 }
 
```

Comment:
Moved scroll handling into common.c so that all apps can use the same logic.  Also updated how scroll is calculated removing the need for MUX.ITEM.PREV_LOW, MUX.ITEM.PREV_HIGH, MUX.ITEM.NEXT_LOW, MUX.ITEM.NEXT_HIGH settings